### ISSUE TITLE
Ordering of assets in the "by asset" output table

### ISSUE THREAD

**Issue by jdebacker** [OpenSourcePolicyCenter/webapp-public#387]
Currently, assets are ordered alphabetically.  Also, assets are groups by the `asset_category` column and not by the `major_asset_group` column.  

Assets should appear in the order:

```
All Assets
 Equipment
  Mainframes
  PCs
  DASDs
  Printers
  Terminals
  Tape drives
  Storage devices
  System integrators
  Communications
  Nonelectro medical instruments
  Electro medical instruments
  Nonmedical instruments
  Photocopy and related equipment
  Office and accounting equipment
  Nuclear fuel
  Other fabricated metals
  Steam engines
  Internal combustion engines
  Metalworking machinery
  Special industrial machinery
  General industrial equipment
  Electric transmission and distribution
  Light trucks (including utility vehicles)
  Other trucks, buses and truck trailers
  Autos
  Aircraft
  Ships and boats
  Railroad equipment
  Household furniture
  Other furniture
  Other agricultural machinery
  Farm tractors
  Other construction machinery
  Construction tractors
  Mining and oilfield machinery
  Service industry machinery
  Household appliances
  Other electrical
  Other
 Structures
  Office                                    
  Hospitals                             
  Special care                          
  Medical buildings                       
  Multimerchandise shopping            
  Food and beverage establishments
  Warehouses                                
  Mobile structures
  Other commercial
  Manufacturing                               
  Electric                                
  Wind and solar
  Gas
  Petroleum pipelines
  Communication                             
  Petroleum and natural gas          
  Mining                                    
  Religious                                 
  Educational and vocational       
  Lodging                                   
  Amusement and recreation         
  Air transportation
  Other transportation
  Other railroad
  Track replacement
  Local transit structures
  Other land transportation                       
  Farm                                      
  Water supply
  Sewage and waste disposal
  Public safety
  Highway and conservation and development
 Inventories
 Land
```

**Comment by PeterDSteinberg** [comment]
@jdebacker  I am getting a key on `Religious` after converting to lower case and comparing to the row labels found in the asset data as lower case:

```
	[u'ships and boats',
 u'mainframes',
 u'nonelectro medical instruments',
 u'own account software',
 u'gas',
 u'warehouses',
 u'general industrial equipment',
 u'light trucks (including utility vehicles)',
 u'equipment',
 u'other',
 u'air transportation',
 u'printers',
 u'electric transmission and distribution',
 u'other electrical',
 u'structures',
 u'household furniture',
 u'other agricultural machinery',
 u'mining and oilfield machinery',
 u'tape drives',
 u'mining',
 u'service industry machinery',
 u'railroad equipment',
 u'office and accounting equipment',
 u'amusement and recreation',
 u'pcs',
 u'other trucks, buses and truck trailers',
 u'terminals',
 u'local transit structures',
 u'autos',
 u'nonmedical instruments',
 u'multimerchandise shopping',
 u'nuclear fuel',
 u'petroleum and natural gas',
 u'construction tractors',
 u'farm tractors',
 u'photocopy and related equipment',
 u'manufacturing',
 u'special care',
 u'public safety',
 u'special industrial machinery',
 u'metalworking machinery',
 u'household appliances',
 u'land',
 u'steam engines',
 u'mobile structures',
 u'residential',
 u'petroleum pipelines',
 u'all investments',
 u'prepackaged software',
 u'other construction machinery',
 u'other commercial',
 u'communications',
 u'storage devices',
 u'inventories',
 u'communication',
 u'other fabricated metals',
 u'internal combustion engines',
 u'system integrators',
 u'educational and vocational',
 u'custom software',
 u'office',
 u'medical buildings',
 u'electro medical instruments',
 u'electric',
 u'other furniture',
 u'track replacement',
 u'water supply',
 u'other land transportation',
 u'wind and solar',
 u'farm',
 u'other transportation',
 u'other railroad',
 u'lodging',
 u'sewage and waste disposal',
 u'aircraft',
 u'hospitals',
 u'highway and conservation and development',
 u'dasds',
 u'food and beverage establishments']
```

**Comment by jdebacker** [comment]
@PeterDSteinberg - ah sorry, I should have removed "Religious" structures from the list above.  We are not computing results for these assets since they do not face federal tax.  Please remove from list (they are already removed from B-Tax output tables - as you are seeing from this error).

**Comment by PeterDSteinberg** [comment]
Fixed by #392

**Comment by jdebacker** [comment]
@PeterDSteinberg, I think we are missing the "Land" category in the output table.  Would you agree?

**Comment by jdebacker** [comment]
Fixed with PR #430
